LocalSettings.php

```diff
@@ -6473,7 +6473,7 @@
 		'worlduniversityandschoolwiki' => "//$wmgUploadHostname/worlduniversityandschoolwiki/6/60/WorldUnivAndSchLogo2017-08-18Wave.png",
 		'zharkunuwiki' => "//$wmgUploadHostname/zharkunuwiki/4/41/Zharkunu_Logo_-_135.png",
 	),
-	'wgLogo' => array(
+	'53' => array(
```

Comment:
seems like a typo, please move back to wgLogo

Reply:
Right. I wonder where did that come from.